src/main/java/gov/cancer/pageobject/components/RadioButton.java

```diff
@@ -8,29 +9,32 @@
 public class RadioButton extends CGovField {
 
   //WebElement of radioButton
- private WebElement radioButton;
+  private WebElement radioButton;
+
   /**
-   *Constructor
+   * Constructor
+   *
    * @param element
    */
-  public RadioButton (WebElement element){
+  public RadioButton(WebElement element) {
     super(element);
-    radioButton= element;
+    radioButton = element;
   }
 
   /**
    * Method is used to check radioButton
    */
-  public void select(){
+  public void select() {
     radioButton.click();
   }
 
   /**
    * Method checks if the radio button is selected
+   *
    * @return
    */
-  public boolean isSelected(){
-    return radioButton.isSelected();
+  public boolean isSelected() {
+    return ElementHelper.findElement(radioButton, ":scope input").isSelected();
```

Comment:
On line 28, `radioButton` is assumed to point directly to the field.
On line 37, an additional selector is used to find the field.

This should be consistent, and if the additional selector, it should be applied in the constructor.  (OTOH, the `WebElement` should be assumed to contain both the button control, and any associated label.)